examples/remote/complicated_cargo_library/Cargo.toml

```diff
@@ -19,6 +19,25 @@ workspace_path = "//remote/complicated_cargo_library/cargo"
 gen_workspace_prefix = "remote_complicated_cargo_library"
 genmode = "Remote"
 incompatible_relative_workspace_path = true
+targets = [
```

Comment:
This should not be necessary, By specifying no targets, all targets should be included. Or did I miss something in this change?